build.rs

```diff
@@ -0,0 +1,38 @@
+use std::process::Command;
+
+use chrono::Utc;
+
+const BUILD_INFO_FALLBACK: &str = "Unknown";
+
+fn run_command(program: &str, args: &[&str]) -> String {
+    if let Ok(o) = Command::new(program).args(args).output() {
+        let raw_output = String::from_utf8_lossy(&o.stdout).into_owned();
+        raw_output.trim().to_string()
+    } else {
+        BUILD_INFO_FALLBACK.to_string()
+    }
+}
+
+fn set_env(var: &str, value: String) {
+    println!("cargo:rustc-env={}={}", var, value);
+}
+
+fn main() {
+    set_env(
+        "TIKV_BUILD_TIME",
+        Utc::now().format("%Y-%m-%d %H:%M:%S").to_string(),
```

Comment:
This won't work, since we don't want to run the build script every time, but if we don't do that the time in the env var will be the last time the build script was run.